frontend/libs/studio-components/src/components/StudioToggleableTextfield/StudioTextfieldToggleView/StudioTextfieldToggleView.tsx

```diff
@@ -18,7 +18,7 @@ export const StudioTextfieldToggleView = ({
     <StudioButton className={className} onClick={onClick} {...rest}>
       <span className={classes.viewModeIconsContainer}>
         <KeyVerticalIcon data-testid='keyIcon' aria-hidden />
-        {children}
+        <div> {children}</div>
```

Comment:
Is the div needer here? 😊

Reply:
It is no longer needed after removing the className, I removed